ref/general/mp-openapi.adoc

```diff
@@ -10,47 +10,80 @@
+
+For example, the following code example shows JAX-RS code to request the list of host names in the inventory. The code has been annotated with OpenAPI annotations to provide more useful and explanatory information about the operation:
+
+[source,java]
+----
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @APIResponse(
+        responseCode = "200",
+        description = "host:properties pairs stored in the inventory.",
+        content = @Content(
+            mediaType = "application/json",
+            schema = @Schema(
+                type = SchemaType.OBJECT,
+                implementation = InventoryList.class)))
+    @Operation(
+        summary = "List inventory contents.",
+        description = "Returns the currently stored host:properties pairs in the "
+        + "inventory.")
+    public InventoryList listContents() {
+        return manager.list();
+    }
+----
+
+In the following OpenAPI documentation example that is generated from the example code, the JAX-RS framework generates the path /inventory/systems based on the value of the @Path annotation, and the value of the operationId key (listContents) based on the name of the method (listContents()). The other parts of the document are generated from the OpenAPI annotations with which the developer annotated the code. For example, the @Operation annotation provides more meaningful information about what the operation does and why; in this case, the operation returns a list of the host names with properties of each host that are currently stored by the inventory web service. This is described by the summary and description keys in the @Operation annotation:
+
+[source,java]
+----
+/inventory/systems:
+  get:
+    summary: List inventory contents.
+    description: Returns the currently stored host:properties pairs in the inventory.
+    operationId: listContents
+    responses:
+      200:
+        description: host:properties pairs stored in the inventory.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/InventoryList'
+----
+
+The information provided through the OpenAPI annotations augment the basic API documentation that is generated by the JAX-RS framework.
+
+For examples of annotations, see link:https://www.openliberty.io/docs/ref/microprofile/3.0/#package=org/eclipse/microprofile/openapi/annotations/package-frame.html&class=org/eclipse/microprofile/openapi/annotations/Components.html[Annotation Samples]
+
+== Manually created REST API documentation
+
+You can generate the static files before any code with an editor like Swagger editor. In this approach, popularly known as the 'design first' approach, the API contract is designed first before writing any code. The API contract acts as the central draft, often referred to as the "source of truth". This approach is considered efficient as it helps to spot issues in the design, before writing any code rather than after the implementation.
```

Comment:
Change "before writing any code" to "before you write any code". We typically want to avoid gerunds when possible to convey a more active voice.
There are two instances of this that I noticed in this section.